src/main/kotlin/step3/domain/car/Car.kt

```diff
@@ -0,0 +1,14 @@
+package step3.domain.car
+
+class Car(val id: Int, private val carMovePointDecider: CarMovePointDecider) {
+
+    private var position: Int = 0
```

Comment:
코틀린은 프로퍼티의 getter/setter에 대한 접근자를 지원합니다!
이를 활용해보면 다음과 같이 표현할 수 있을 것 같아요.
```suggestion
class Car(
    val id: Int,
    private val carMovePointDecider: CarMovePointDecider,
    position: Int = 0
) {

    var position: Int = position
        private set
```